src/lib/helpers/random-id.ts

```diff
@@ -0,0 +1,11 @@
+import windowExists from './window-exists';
+
+const randomId = (): string => {
+  if (windowExists()) {
+    return window.crypto.getRandomValues(new Uint32Array(1))[0].toString(16);
+  } else {
```

Comment:
u don't need the `else` statement

Reply:
Yeah, I know that haha. I will change it if this style is preferred. I don't really think you gain much from a readability perspective, personally, which is why I don't do that.